docs/src/increase-observability/metrics/dedicated-generation-3.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Monitor Dedicated Generation 3 metrics
+sidebarTitle: Dedicated Generation 3 metrics
+description: Understand how to read metrics for Dedicated Genration 3 environments.
+---
+
+Dedicated Generation 3 environments consist of:
```

Comment:
Replace  Dedicated Generation 3 environments consist of: with Dedicated Generation 3 environments consists of containers, ran across dedicated Virtual Machines instances: